test/unit/concerns/symbolize_test.rb

```diff
@@ -42,6 +42,8 @@ def test_period_previous_change
   end
 
   test '#changes is symbolized but not #previous_changes' do
+    travel_to Time.now # avoid updated_at in changes on some databases
```

Comment:
A couple of suggestions:

- There's a particular method to preform a `travel_to Time.now`: `freeze_time`
- You can move the test logic to inside a block as `freeze_time do [test code] end` this way you can be sure the time is back to normal after the test. It's not really needed because Rails restores normal time after every test automatically, bu it's clearer.

Reply:
Hm... that's interesting.
I fail to understand how this prevents the `updated_at` field, I guess this is some internal ActiveRecord logic...

Reply:
`updated_at` is updated whenever a record was updated. With MySQL resolution seems to be seconds so it doesn't really change during run. But with oracle and postgres apparently it is something smaller and test fails. Freezing the time prevents `updated_at` to be actually updated so later, comparing he changed fields is not broken.

wrt `freeze_time` I don't see why we should put everything in a block. It hurts the eyes and tear_down should take care of it.

Reply:
> Freezing the time prevents `updated_at` to be actually updated

I can't find why this is so... 
It works, but it does not seem logical to me :upside_down_face: 
Even if the time freezes (i.e. `Time.now` always returns the same), if the value changes, I would expect `updated_at` to be set (even if it would be always set to the same value).

Reply:
If it is set to the same value, then active record does not report it as changed. I think this behavior is like that since Rails 4.x

Reply:
Not only for `updated_at`. If you update an attribute to something that `old_value == new_value` it will not report it as changed.

Reply:
@jlledom alternatively I can make the freeze method to just freeze if block was not provided. So that we have better consistency between tests. WDYT?

If you agree please approve this, I want to merge it and then do this plus a flaky test.

Reply:
> If you update an attribute to something where old_value == new_value it will not report it as changed.

Well, this makes total sense.

> If it is set to the same value, then active record does not report it as changed.

But this does not for me :) But well, now I know. Thanks!

Reply:
I would also prefer to replace `travel_to Time.now` with `freeze_time` (doesn't need to be in a block).